components/molecule/photoUploader/src/ThumbCard.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import AtomIcon, {ATOM_ICON_SIZES} from '@s-ui/react-atom-icon'
+
+import {BASE_CLASS_NAME} from './config'
+const THUMB_CARD_CLASS_NAME = `${BASE_CLASS_NAME}-thumbCard`
+const IMAGE_THUMB_CARD_CLASS_NAME = `${THUMB_CARD_CLASS_NAME}-image`
+const ICON_THUMB_CARD_CLASS_NAME = `${THUMB_CARD_CLASS_NAME}-iconContainer`
+
+const ThumbCard = ({
+  iconSize = ATOM_ICON_SIZES.small,
+  callbackDeleteItem,
+  callbackRetryUpload,
+  callbackRotateItem,
+  deleteIcon,
+  index,
+  image,
+  mainPhotoLabel,
+  rejectPhotosIcon,
+  retryIcon,
+  rotateIcon
+}) => {
+  const hasErrors = image.hasErrors
+
+  const counterClass = cx(`${THUMB_CARD_CLASS_NAME}-counter`, {
+    [`${THUMB_CARD_CLASS_NAME}-mainCounter`]: index === 0
+  })
+
+  return (
+    <div className="sui-MoleculePhotoUploader-thumbCard">
```

Comment:
You should use the const `THUMB_CARD_CLASS_NAME`
```suggestion
    <div className={THUMB_CARD_CLASS_NAME}>
```

Reply:
👍